### PR TITLE
Presentation open source LLMs

### PR DESCRIPTION
**Pull request by ka161heg**

### CODE REVIEW

File: src/htwgnlp/preprocessing.py
Comment (pkeilbach) [inline src/htwgnlp/preprocessing.py:1]: please do not include any assignment code in your PR

File: docs/presentations/articles/assets/image-6.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-6.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/assets/image-3.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-3.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/assets/image-2.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-2.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/assets/image-12.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-12.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/assets/image-11.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-11.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/assets/image-10.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/image-10.png:1]: maybe an image like this one can be included from a public source?

File: docs/presentations/articles/OS_LLMs_Robert.md
Comment (pkeilbach) [inline docs/presentations/articles/OS_LLMs_Robert.md:1]: would it be OK for you to provide the article in [English](https://pkeilbach.github.io/htwg-practical-nlp/presentations/presentations/#blog-article)? Sorry, I'm not sure if I mentioned this in lecture. Of course, it's totally acceptable to use a translation tool (please proof read it though)

**Review by pkeilbach — CHANGES_REQUESTED** [review]
Congrats on the first external PR in this repo, @ka161heg 🚀 🤩

Please see my comments and let me know if there are any questions! 

I also added a naming convention section for the filenames just now (#60), could you please adjust that?

File: src/htwgnlp/preprocessing.py
Comment (ka161heg) [reply src/htwgnlp/preprocessing.py:1]: Sorry, im not very familiar with git, so i didnt knewhow to do it properly.

File: docs/presentations/articles/assets/image-6.png
Comment (ka161heg) [reply docs/presentations/articles/assets/image-6.png:1]: included source under the image because i could not find proper URLS for the images.

**Pull request closed without merging**